dart/common/AddonManager.h

```diff
@@ -86,10 +68,21 @@ class AddonManager
   using Properties = ExtensibleMapHolder<PropertiesMap>;
 
   using AddonMap = std::map< std::type_index, std::unique_ptr<Addon> >;
+  using RequiredAddonSet = std::unordered_set<std::type_index>;
 
   /// Virtual destructor
   virtual ~AddonManager() = default;
 
+  /// Default constructor
+  AddonManager() = default;
+
+  /// It is currently unsafe to copy an AddonManager
+  // TODO(MXG): Consider making this safe by cloning Addons into the new copy
+  AddonManager(const AddonManager&) = delete;
+
+  /// It is currently unsafe to move an AddonManager
+  AddonManager(AddonManager&&) = delete;
```

Comment:
Agreed. Eventually I'd like all of these functions to work by doing a deep copy using ``cloneAddon()``, but for now I think we should just delete them.